math/benches/msm.rs

```diff
@@ -0,0 +1,124 @@
+use criterion::{criterion_group, criterion_main, BenchmarkId, Criterion};
+use lambdaworks_math::cyclic_group::IsCyclicBilinearGroup;
+use lambdaworks_math::elliptic_curve::traits::HasDistortionMap;
+use lambdaworks_math::msm::naive::Naive;
+use lambdaworks_math::{
+    elliptic_curve::{
+        curves::test_curve::{QuadraticNonResidue, ORDER_P},
+        element::EllipticCurveElement,
+        traits::HasEllipticCurveOperations,
+    },
+    field::{
+        element::FieldElement, fields::u64_prime_field::U64PrimeField,
+        quadratic_extension::QuadraticExtensionField,
+    },
+    msm::{pippenger, MSM},
+};
+use rand::rngs::StdRng;
+use rand::{Rng, SeedableRng};
+
+type FE = FieldElement<U64PrimeField<ORDER_P>>;
+
+macro_rules! bench_msm_for_length {
+    ( $func:ident, $a:expr ) => {
+        pub fn $func(c: &mut Criterion) {
+            let msm_size = $a;
+            let window_sizes = vec![1, 2, 3, 4];
+
+            let mut rng = StdRng::seed_from_u64(42);
+            let g = EllipticCurveElement::<CurrentCurve>::generator();
+
+            // Generate Field Elements and Group elements.
+            let s: Vec<FE> = (0..msm_size).map(|_| FE::new(rng.gen())).collect();
+            let hiding: Vec<EllipticCurveElement<CurrentCurve>> = (0..msm_size)
```

Comment:
I'm using the same field element and test curves as we have today, but in the future we can expand the macro also to receive which fields and groups we want to benchmark (e.g: bls12-381, etc) and make a wider set of benchmarks in those dimensions.